android-interop-testing/README.md

```diff
@@ -0,0 +1,46 @@
+gRPC Android test App
+=======================
+
+Implements gRPC integration tests in an Android App.
+
+TODO(madongfly) integrate this App into the gRPC-Java build system.
+
+In order to build this app, you need a local.properties file under this directory which specifies
+the location of your android sdk:
+```
+sdk.dir=/somepath/somepath/sdk
+```
+
+Connect your Android device or start the emulator:
+```
+$ ./start-emulator.sh <AVD name> & ./wait-for-emulator.sh
```

Comment:
Should & be && ? I know we are waiting for it to start, but it seems if the AVD name is wrong or similar wait-for-emulator shouldn't run.